pom.xml

```diff
@@ -251,6 +254,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.]]>
 			<artifactId>commons-io</artifactId>
 			<version>2.11.0</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-text</artifactId>
+			<version>1.10.0</version>
+		</dependency>
```

Comment:
Can you describe why this was added?

Reply:
@jmcneil86043 Correct me if I'm wrong here but Phil mentioned that you all noticed that even release/2022.4.x was affected by this?  I wonder if this is a dependency that Jchem pulls in that only affects jchem builds of acas-roo-server.  @philippcheung One way you can check this is by running `man dependency:tree` and viewing the output for both indigo and jchem (without this change).  My bet is that you see `org.apache.commons` in the jchem build and not the indigo build.  If that is the case, we should move this dependency to inside the jchem profile of the pom.xml

Reply:
Just to clarify what I meant, and this might be in agreement with what you're hypothesizing -- my new version of the roo (before I added this) caused a security trigger, while stage machines didn't trigger the security alert -- so yes, it might be the jchem.

Reply:
Yeah, so I removed the fix above and I see this in the depdendency tree:

```
[INFO] +- com.chemaxon:jchem-main:jar:22.12.0:compile
...etc.
[INFO] |  +- com.chemaxon:descriptors-jep:jar:22.12.0:compile
[INFO] |  |  |  +- com.chemaxon:io-all:jar:22.12.0:compile
[INFO] |  |  |  |  +- com.chemaxon:io-csv:jar:22.12.0:compile
[INFO] |  |  |  |  |  \- org.apache.commons:commons-text:jar:1.9:compile
```
 Which means that the issue is coming from the jchem jar.  I just moved the commons text under the jchem profile so it only gets pulled into the build when doing a jchem build.